src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -42,9 +43,13 @@ def _raise_error_if_not_drawing_classifier_input_sframe(
     if len(dataset) == 0:
         raise _ToolkitError("Input Dataset is empty!")
 
-def create(input_dataset, target, feature=None, 
+def print_validation_track_notification():
```

Comment:
I'm not really understanding why this is its own function. It should probably be a hidden function.